src/app/route/index.js

```diff
@@ -216,13 +224,15 @@ class AppRouter extends PureComponent {
     renderDefaultRouterContent() {
         return (
             <>
-                { this.renderItemsOfType(BEFORE_ITEMS_TYPE) }
-                <NoMatchHandler>
-                    <Switch>
-                        { this.renderItemsOfType(SWITCH_ITEMS_TYPE) }
-                    </Switch>
-                </NoMatchHandler>
-                { this.renderItemsOfType(AFTER_ITEMS_TYPE) }
+                <Suspense fallback={ null }>
```

Comment:
Maybe add a function call here, and return null. If someone would decided to use this loader, it will be easy for him to do.